src/headers/agent_op.h

```diff
@@ -84,12 +84,55 @@ int auth_connect();
 // Close socket if valid.
 int auth_close(int sock);
 
-// Send a local agent add request.
-int w_request_agent_add_local(int sock, char *id, const char *name, const char *ip, const char * groups, const char *key, const int force, const int json_format, const char *agent_id, int exit_on_error);
+/**
+ * @brief Send a local agent add request.
```

Comment:
I would highlight the type of request. Something like... `Send a local agent "add" request.`

Reply:
Done